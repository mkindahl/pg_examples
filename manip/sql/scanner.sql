CREATE EXTENSION manip;

CREATE TABLE job (job_id serial, body regproc, descr text);

INSERT INTO job(body,descr) VALUES
       ('scan_table', 'Scan table'),
       ('pg_backend_pid', 'Get backend PID');
       
CALL scan_table('job');
