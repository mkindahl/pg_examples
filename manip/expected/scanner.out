CREATE EXTENSION manip;
CREATE TABLE job (job_id serial, body regproc, descr text);
INSERT INTO job(body,descr) VALUES
       ('scan_table', 'Scan table'),
       ('pg_backend_pid', 'Get backend PID');
       
CALL scan_table('job');
NOTICE:  attnum: 1, attname: job_id, value: 1
NOTICE:  attnum: 2, attname: body, value: scan_table
NOTICE:  attnum: 3, attname: descr, value: Scan table
NOTICE:  attnum: 1, attname: job_id, value: 2
NOTICE:  attnum: 2, attname: body, value: pg_backend_pid
NOTICE:  attnum: 3, attname: descr, value: Get backend PID
